lib/spack/external/argparse.py

```diff
@@ -108,8 +94,6 @@
 import sys as _sys
 import textwrap as _textwrap
 
-from llnl.util.tty.colify import colified
```

Comment:
It looks like we had slightly modified the vendored copy of argparse. Is this still needed? In general, I'm not too keen on modifying vendored packages, as it makes it more difficult to upgrade.

Reply:
Actually, it looks like we've modified argparse several times:
https://github.com/spack/spack/commits/develop/lib/spack/external/argparse.py

@tgamblin Can you comment?